js/app/travelwidget.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * A module which defines the SalaryWidget
+ * @module app/travelwidget
+ */
+define(["jquery", "app/widget", "app/travelitem-hotel",
+    "app/travelitem-car", "app/travelitem-plane"],
+function(jquery, Widget, TravelItemHotel, TravelItemCar, TravelItemPlane){
```

Comment:
Requiring travelitem-car and plane without those files prevents this widget from loading, please remove these requirements until you can commit those files